src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,19 @@
+const ArticleCard = (props) => {
+    const { articleObj } = props
+    const { article_img_url, article_id, title } =  articleObj 
+
+    console.log(articleObj)
+
+    return (
+        <div className="ArticleCard">
+            <img src={article_img_url} alt="test"/>
+               <div className="CardBanner">
+                    <h2>
+                        {title}
```

Comment:
we also want to see some more information per article, such as the number of votes and comments it has, the author and the date it was posted